react/components/ProductPrice/index.js

```diff
@@ -15,13 +16,24 @@ class Price extends Component {
       sellingPrice,
       listPrice,
       installments,
-      installmentPrice,
       showInstallments,
       showLabels,
       showSavings,
       intl: { formatNumber },
     } = this.props
 
+    const noInterestRateInstallments = installments.filter(
+      installment => !installment.InterestRate
+    )
+
+    const installment = (isEmpty(noInterestRateInstallments)
```

Comment:
just something minor, but could you add a comment briefly explaining this logic?

Reply:
Ok